pkgs/development/tools/build-managers/buildbot/8.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv, buildPythonApplication, fetchurl, twisted, dateutil, jinja2
```

Comment:
that is 8.nix, which is being left intact as it was (previously default.nix).   

i was planning to rebase https://github.com/NixOS/nixpkgs/pull/17761 (bb8) off this once merged.